.github/workflows/issue-for-config.yml

```diff
@@ -0,0 +1,58 @@
+name: Check PR for configuration changes
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+    - 'features/featureflag_string.go'
+    - 'test/config-next/*.json'
+    - 'test/config-next/*.yaml'
+    - 'test/config-next/*.yml'
+
+jobs:
+  check-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Comment PR
+        id: comment-pr
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const commentMarker = '<!-- deployment_ticket_check -->';
+            const prAuthor = context.payload.pull_request.user.login;
+            const commentBody = `${commentMarker}\n@${prAuthor}, this PR appears to contain configuration changes. Please ensure that a corresponding deployment ticket has been filed with the new configuration values.\n`;
+            const { owner, repo, number: issue_number } = context.issue;
+
+            // Define the JIRA issue number pattern.
+            const issueRegex = /IN-\d+/;
+
+            // Get PR body and all issue comments.
+            const prBody = context.payload.pull_request.body;
+            const comments = await github.rest.issues.listComments({
+              owner,
+              repo,
+              issue_number
+            });
+
+            // Check if the PR body or any comment includes a JIRA issue number.
+            if (issueRegex.test(prBody) || comments.data.some(c => issueRegex.test(c.body))) {
+              return;
+            }
+
+            // Check if our comment already exists.
+            const ourComment = comments.data.find(c => c.body.includes(commentMarker));
```

Comment:
nit: no need to predeclare this because it's only used once; inline this into the conditional on line 47.